source/import-export.txt

```diff
@@ -109,8 +120,7 @@ Export Data from a Collection
 
 |compass| can export data from a collection as either a
 **JSON** or **CSV** file. If you specify a
-:ref:`filter <query-bar-filter>` in the
-:ref:`query bar <compass-query-bar>` prior to export, |compass-short|
+:ref:`filter <query-bar-filter>`, |compass-short|
```

Comment:
I'd reword this a bit to say: "Compass only exports documents which match the specified query." With the new flow, you're explicitly specifying an export filter, so I think we can be more direct.